tests/api/v1/bots/delete.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/api/v1/bots/delete.js
```

Comment:
I see that you are not using token for any tests and most of the CRUD tests will fail if we enforce token in test environment. It would be great if you can include tokens in test so that we don't have to change them in future (when we plan to enforce token by default). Does that makes sense?